src/Locale/de/cake.po

```diff
@@ -246,7 +246,7 @@ msgstr "Start"
 
 #: View/Widget/DateTimeWidget.php:540
 msgid "January"
-msgstr "Januar"
+msgstr "Jänner"
```

Comment:
But dialects are a complicated thing...

Anyway, according to [Wikipedia](https://de.wikipedia.org/wiki/Januar#J%C3%A4nner):

> Selbst im Süden, in Bayern und in der Schweiz, wird Jänner schriftsprachlich nur noch selten gebraucht.

In anderen Teilen des oberdeutschen Sprachraums, so in Österreich und Südtirol, ist hingegen Jänner nach wie vor die allgemein übliche Bezeichnung für den ersten Monat des Jahres, sowohl umgangs- und schriftsprachlich als auch nach dem Österreichischen Wörterbuch. 

Let's check it with a (seemingly) popular Austrian news paper:
Januar yields 3'180 results
https://www.google.ch/?gws_rd=ssl#q=site:http:%2F%2Fwww.krone.at%2F+januar r 

Jänner yields 32'400 results
https://www.google.ch/?gws_rd=ssl#q=site:http:%2F%2Fwww.krone.at%2F+j%C3%A4nner

I guess I was wrong...

@mdlouhy so that's a 👍 for using "Jänner" in a ``de_AT`` locale from me 😸